webapp/src/containers/SubscribeModal/index.tsx

```diff
@@ -102,6 +105,12 @@ const SubscribeModal = () => {
         resetModalState();
     };
 
+    // Opens subscription modal
+    const handleSubscriptionModal = () => {
+        resetModalState();
+        dispatch(toggleShowSubscribeModal({isVisible: true, commandArgs: []}));
```

Comment:
```suggestion
        dispatch(toggleShowSubscribeModal({isVisible: true, commandArgs: []}));
        resetModalState();
```

always do `        dispatch(toggleShowSubscribeModal({isVisible: true, commandArgs: []}));` as first thing to prevent seeing other reset action happens while modal is about to close